pkg/network/http/ebpf_gotls.go

```diff
@@ -109,7 +145,25 @@ func newGoTLSProgram(c *config.Config) *GoTLSProgram {
 		log.Errorf("System arch %q is not supported for goTLS", runtime.GOARCH)
 		return nil
 	}
-	return &GoTLSProgram{}
+	p := &GoTLSProgram{
+		procRoot: c.ProcRoot,
+		analysisCache: *ttlcache.New(
+			ttlcache.WithTTL[inodeNumber, *hookedBinary](2*time.Minute),
+			ttlcache.WithDisableTouchOnHit[inodeNumber, *hookedBinary](),
```

Comment:
what does that mean?

Reply:
By default, ttlcache resets the TTL when an entry is accessed. This option disables that.
This is an artifact of an earlier version I've wrote, and is not really needed anymore. It might even make new processes handling a bit slower in a very specific case, so I'll remove it.